cli/processexecutor.cpp

```diff
@@ -310,19 +311,19 @@ unsigned int ProcessExecutor::check()
         if (!rpipes.empty()) {
             fd_set rfds;
             FD_ZERO(&rfds);
-            for (std::list<int>::const_iterator rp = rpipes.cbegin(); rp != rpipes.cend(); ++rp)
+            for (auto rp = rpipes.cbegin(); rp != rpipes.cend(); ++rp)
```

Comment:
this should be a range for loop right?

Reply:
It probably can - but so do others. `modernize-loop-convert` is currently disabled and that should report by it. Now that we have made more cleanups I will give that another spin.

Reply:
PR enabling that warning is coming up after this.